src/NzbDrone.Core/ImportLists/LastFm/LastFmApi.cs

```diff
@@ -0,0 +1,21 @@
+
+using System.Collections.Generic;
+
+namespace NzbDrone.Core.ImportLists.LastFm
+{
+    public class LastFmArtistList
+    {
+        public List<LastFmArtist> Artist { get; set; }
+    }
+
+    public class LastFmArtistResponse
+    {
+        public LastFmArtistList Topartists { get; set; }
```

Comment:
To keep formatting that would likely be better, I don’t know if it’ll have an effect on deserialization of the api json though since that’s in LC. I’ll give it a try